dist/robotframework/res/philip.keywords.txt

```diff
@@ -4,12 +4,6 @@ Library             PhilipAPI  port=%{PHILIP_PORT}  baudrate=${115200}  WITH NAM
 Resource            riot_base.keywords.txt
 
 *** Keywords ***
-Reset DUT and PHILIP
```

Comment:
So we are not resetting philip now?
Maybe the script that does the reset should be included.

Reply:
oh we do, but in a HIL deployment specific reset script that is triggered by `make reset` and configured using environment variables.

I'll add the scripts after some more tests.